### DISCUSSION TITLE
admonition list within a list

### DISCUSSION THREAD

**Discussion by digihernandez** [squidfunk/mkdocs-material#6290] [Q&A]
1. Bring your necessities:

        !!! warning
             - Do not bring any alcoholic beverages
             - Do not bring cigarettes or vape
             - Do not talk to strangers

2. Avoid nuisances

Please help me. I can't figure out how to do that in MKDOCS. An admonition list within a list. The numbering will start again after I put the admonition list. Please kindly help me.

**Comment by facelessuser** [comment]
Because of how you inserted the content into GitHub (not in a code block) I can't see how you formatted your Markdown, but you need to indent content under lists and such 4 spaces:

```md
1. Bring your necessities:

    !!! warning
        - Do not bring any alcoholic beverages
        - Do not bring cigarettes or vape
        - Do not talk to strangers

2. Avoid nuisances
```

<img width="810" alt="Screenshot 2023-11-04 at 9 21 31 AM" src="https://github.com/squidfunk/mkdocs-material/assets/1055125/a7d73d7e-8aa5-47c4-9dbd-520a259fb7c7">

**Reply by digihernandez** [reply]
When I copy the way your whole text and paste it into my docu, it works.   But when I do it on my own, it doesn't. 
But thank you very much :) It really helped me.

**Reply by facelessuser** [reply]
The important thing to remember is that when indentation is required, in any syntax, it must be 4 spaces. There may be extensions that alter this behavior, and if so, you will have to read their documentation to understand the corner cases etc.
